DigitalLearningSolutions.Data.Migrations/DigitalLearningSolutions.Data.Migrations.csproj

```diff
@@ -5,6 +5,10 @@
     <Nullable>enable</Nullable>
   </PropertyGroup>
 
+  <ItemGroup>

```

Comment:
I'm not sure why this has  been added. It seems to reference a path on your local machine.